### PR TITLE
Allow eth_gasPrice to accept param

### PR DESCRIPTION
**Pull request by aaronmgdr**
when using celo eth_gasPrice accepts an optional param of a token address and returns price of gas denominated in that token. for testing need to be able to call the method like that

### CODE REVIEW

**Comment by aaronmgdr** [comment]
note not sure if i got the deps tree all updated

File: src/packages/cli/package.json
Comment (soloseng) [inline src/packages/cli/package.json:1]: update `"@celo/ganache-ethereum": "0.9.0-unofficial.0"` in `devDependencies`

Comment (aaronmgdr) [reply src/packages/cli/package.json:1]: thanks

**Review by soloseng — APPROVED** [review]
LGTMA

**Pull request merged**